<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketIMPWidget</class>
 <widget class="QWidget" name="SocketIMPWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocketIMPWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="cState">
         <property name="text">
          <string>连接状态</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>服务器IP地址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cAddressEdit">
           <property name="placeholderText">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>服务器端口号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cPortEdit">
           <property name="placeholderText">
            <string>5666</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="cMessageHistory"/>
       </item>
       <item>
        <widget class="QLineEdit" name="cMessageEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
